siad/gateway_test.go

```diff
@@ -43,7 +43,7 @@ func TestPeering(t *testing.T) {
 	// Create to peers and add the first to the second.
 	peer1 := newDaemonTester(t)
 	peer2 := newDaemonTester(t)
-	peer1.callAPI("/gateway/peer/add?addr=" + string(peer2.netAddress()))
+	peer1.callAPI("/gateway/peer/add?Address=" + string(peer2.netAddress()))
```

Comment:
we are stuck between two standards. One is the way that the web typically works and the other is the way that Go typically works.

I guess this is slightly different because it's not in json, it's in a url. For json, I would definitely defer to go but here we're doing the conversion anyway.

I guess I'll make it all lowercase.